_data/projects/shellrb.yml

```diff
@@ -8,5 +8,5 @@ tags:
 - Shell
 
 upforgrabs:
-  name: Help Wanted
+  name: Up For Grabs
   link: https://github.com/faraazahmad/shellrb/labels/help%20wanted
```

Comment:
The "help wanted" label on the issue tracker still exists. Is this intentional, or should it also be renamed to "up for grabs"?

Reply:
I'm renaming all to up for grabs

Reply:
👍 then you'll need to change the `link` value below when that's been updated on the issue tracker